### PR TITLE
Format fixing for PowerFlex

### PR DESCRIPTION
**Pull request by shanmydell**
# Description
Format fixing for PowerFlex

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/352 |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [ ] Have you tested the changes locally?
- [ ] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/csidriver/installation/helm/powerflex.md [hunk 1]
```diff
@@ -141,24 +141,22 @@ kubectl -n kube-system kustomize deploy/kubernetes/snapshot-controller | kubectl
 ```
  *NOTE: To use multiple arrays, copy and paste section above for each array. Make sure isDefault is set to true for only one array.* 
 
-
-    After editing the file, run the following command to create a secret called `vxflexos-config`:
+After editing the file, run the following command to create a secret called `vxflexos-config`:
```
Comment (rsedlock1958) [inline content/docs/csidriver/installation/helm/powerflex.md:144]: Replace "following" with "this"

File: content/docs/csidriver/installation/helm/powerflex.md [hunk 2]
```diff
@@ -141,24 +141,22 @@ kubectl -n kube-system kustomize deploy/kubernetes/snapshot-controller | kubectl
...
+- Old `json` format of the array configuration file is still supported in this release. If you already have your configuration in `json` format, you may continue to maintain it or you may transfer this configuration to `yaml`format and replace/update the secret.  
+- "insecure" parameter has been changed to "skipCertificateValidation" as insecure is deprecated and will be removed from use in config.yaml or secret.yaml in a future release. Users can continue to use any one of "insecure" or "skipCertificateValidation" for now. The driver would return an error if both parameters are used.
+- Please note that log configuration parameters from v1.5 will no longer work in v2.0 and higher. Please refer to the [Dynamic Logging Configuration](../../../features/powerflex#dynamic-logging-configuration) section in Features for more information.
+- If the user is using complex K8s version like "v1.21.3-mirantis-1", use below kubeVersion check in helm/csi-unity/Chart.yaml file.
```
Comment (rsedlock1958) [inline content/docs/csidriver/installation/helm/powerflex.md:159]: Replace "below" with "this"

**Review by rsedlock1958 — COMMENTED** [review]
I have reviewed this and suggested minor editorial revisions.

File: content/docs/csidriver/installation/helm/powerflex.md [hunk 3]
```diff
@@ -141,24 +141,22 @@ kubectl -n kube-system kustomize deploy/kubernetes/snapshot-controller | kubectl
...
  *NOTE: To use multiple arrays, copy and paste section above for each array. Make sure isDefault is set to true for only one array.* 
 
-
-    After editing the file, run the following command to create a secret called `vxflexos-config`:
+After editing the file, run the following command to create a secret called `vxflexos-config`:
     
     `kubectl create secret generic vxflexos-config -n vxflexos --from-file=config=samples/config.yaml`
 
-    Use the following command to replace or update the secret:
+Use the following command to replace or update the secret:
```
[the remark this answers is not in the document]
Comment (shanmydell) [reply content/docs/csidriver/installation/helm/powerflex.md]: addressed

**Comment by shanmydell** [comment]
> I have reviewed this and suggested minor editorial revisions.

addressed the comments

**Review by JacobGros — APPROVED** [review]

**Review by prablr79 — APPROVED** [review]

**Pull request merged**